pkg/controller/master/rancher/harvester_node_driver.go

```diff
@@ -0,0 +1,43 @@
+package rancher
+
+import (
+	rancherv3api "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	"github.com/sirupsen/logrus"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	harvesterDriverName = "harvester"
+	driverURL           = "https://github.com/harvester/docker-machine-driver-harvester/releases/download/v0.1.0/docker-machine-driver-harvester-amd64.tar.gz"
+	driverUIURL         = "https://github.com/harvester/ui-driver-harvester/releases/download/v0.1.0/component.js"
+)
+
+func (h *Handler) AddHarvesterNodeDriver() (*rancherv3api.NodeDriver, error) {
```

Comment:
Can we add upgrade logic for bumping the driver in the future?

Reply:
done